server/runtime_javascript_init.go

```diff
@@ -58,9 +58,9 @@ type jsMatchHandlers struct {
 }
 
 type RuntimeJavascriptCallbacks struct {
-	Rpc                            map[string]string
-	Before                         map[string]string
-	After                          map[string]string
+	Rpc                            *MapOf[string, string]
```

Comment:
From the docs:
```
// The MapOf type is optimized for two common use cases: (1) when the entry for a given
// key is only ever written once but read many times, as in caches that only grow,
// or (2) when multiple goroutines read, write, and overwrite entries for disjoint
// sets of keys. In these two cases, use of a MapOf may significantly reduce lock
// contention compared to a Go map paired with a separate Mutex or RWMutex.
```
I guess this would fall into (1), but I can change it anyway if you think it's best